.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
   - conda info -a
 
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy nose pip matplotlib coverage
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION 'numpy <1.15.3' scipy nose pip matplotlib coverage
```

Comment:
I am a little worried about pinning the version number of a major package like this, but if we have no choice...